node/src/bin/manager.rs

```diff
@@ -445,6 +442,36 @@ pub enum IndexCommand {
     },
 }
 
+#[derive(Clone, Debug, StructOpt)]
+pub struct FixBlockCommand {
+    /// Network name (must fit one of the chain)
+    #[structopt(empty_values = false)]
+    network_name: String,
+    #[structopt(subcommand)]
+    method: FixBlockSubCommand,
+}
+
+#[derive(Clone, Debug, StructOpt)]
+enum FixBlockSubCommand {
+    /// The number of the target block
+    ByHash { hash: String },
+
+    /// The hash of the target block
+    ByNumber { number: i32 },
+
+    /// A block number range in the form: `start..end`
+    ///
+    /// All valid [`std::ops::Range`] definitions are accepted.
```

Comment:
This requires that users know the Rust syntax for ranges. Would be better to spell that out. Or just have one allowed syntax. A third option is to pass in separate `--from` and `--to` arguments instead of a literal range.

Reply:
I agree. 
Would you think it would make sense if we ask for confirmation for open-ended intervals? 
For instance, if the command is invoked with only `--from` or `--to`, we could bring a prompt to confirm the operation.

In that case, we should also provide a `--no-confirm` flag to auto-confirm those prompts.

Reply:
Fixed in 180124537